image-index.md

```diff
@@ -83,6 +83,8 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
         This property is RESERVED for future versions of the specification.
 
+  If multiple manifests match a client or runtime's requirements, the first matching entry SHOULD be used.
```

Comment:
```suggestion
  If multiple manifests match a client or runtime's `platform` requirements, the first matching entry SHOULD be used.
```

Reply:
I was intentionally leaving that out since some clients may have different requirements, like checking annotations to see if a different layer compression format is available.